WEBDAV.md

```diff
@@ -4,6 +4,8 @@ The Internxt CLI comes with built in WebDav support.
 
 ## How it works
 
+When you login with Internxt CLI, your session and some other configuration data are stored in your local home folder. These data will be used for both the CLI and WebDAV. Restarting the computer or uninstalling the CLI will not erase this configuration. The only way to properly clean up this configuration is by explicitly using the logout command.
```

Comment:
session sounds too generic, can we be more specific about what we store?

- auth tokens
- your decrypted mnemonic